software-engineer/en_US/01.coding-skills.md

```diff
@@ -1,3 +1,30 @@
 # Coding Skills
 
-English version is under constuction. Please see the [Portuguese version](https://github.com/loft-br/study-guide-hiring-process/tree/coding-skills-revamp/software-engineer/pt_BR/01.coding-skills.md) for the time being.
+In this interview, we'd like to assess your skills in making business requisites into __readable code__.
+Think of it as a normal pair programming dynamic, that we do on our day-to-day.
+You'll be the one conducting this pairing, but we want you to be at ease to discuss any ideas and doubts that you might have with the interviewer(s). Also feel free to use any online search tool and your favorite IDE.
+The only thing we ask is that the exercise is done while sharing the screen with the interviewers, so that it's possible to keep up with your line of thought.
+
+The code should be initialized from scratch, in the programming language of your preference.
+It will be a simple command line script, without any intention of ever evolving to an API (backend) or a HTML page (frontend).
+
+__Bringing a blank project setup might save you a few minutes__. This means a project capable to print an arbitrary string on the console, with the main packages/libraries that you feel like using already referenced. If you'd like to use automated tests, it's also a goog idea to previously setup the framework to do so.
+
+You can also code this exercise using an online coding tool, called Codility.
+
+## Main evaluation criteria
+* Code running and executing the challeng premises.
+* Readable code
+  * Coherent naming for variables and functions
+  * Adequate responsibility separation
+  * Ease of understanding code behavior
```

Comment:
achei meio estranho esse phrasing mas não pensei em nada melhor ainda xD

Reply:
sim, também me pareceu um pouco estranho ele, mas não consegui pensar em nada melhor tb... =(

Reply:
talvez "Code behavior's ease of understanding"?